src/main/java/managers/WebDriverManager.java

```diff
@@ -0,0 +1,39 @@
+package managers;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import java.util.Locale;
+
+public class WebDriverManager {
+    public WebDriverManager(String browserType) {
+        this.browserType = browserType;
+    }
+
+
+    private WebDriver activeWebDriver;
+
+
+    private String browserType;
+
+    private void initializeDriver() {
+        if (browserType.toUpperCase(Locale.ROOT).equals("Chrome")) {
```

Comment:
the execution will fail because you are comparing the UPPER CASE TEXT with Chrome instead of CHROME. Please change Chrome with CHROME